### ISSUE TITLE
Events not shown in Stats

### ISSUE THREAD

**Issue by alangbha** [getsentry/self-hosted#2272]
### Self-Hosted Version

23.3.1

### CPU Architecture

x86_64

### Docker Version

24.0.4

### Docker Compose Version

1.24.0

### Steps to Reproduce

1. Go to **Stats** (left panel)

### Expected Result

The graph should show all events that were created

### Actual Result

The graph only shows some events even when they are seen on issues

![image](https://github.com/getsentry/self-hosted/assets/94523389/a916b23c-063e-4517-87ee-d0f58e61a4ec)

I am not sure if this is actually an error or just visualization problem.
I do not have an initial point of where to look for errors or which services this is related to.
Issues are reported properly and they are seen in the issue list. Also, alerts work.

**Comment by azaslavsky** [comment]
Is it possible that there were spikes in errors? Ex, that you had an outage or something similar around Jul 2-3 and a smaller one around Jul 11, and that otherwise your baseline error rate is very low? You could try setting the date range filer for a shorter period of time where there are no spikes, and see if the vertical scale on the graph zooms in to show you the smaller error rate.

**Comment by alangbha** [comment]
Values of Accepted/Filtered/Dropped errors are really 0. This is now for example for the last 7 days
![image](https://github.com/getsentry/self-hosted/assets/94523389/96daaddc-e4ea-4892-912b-3430071241e2)

**Comment by azaslavsky** [comment]
Do you have logs from one of the periods when errors stopped coming in? I wonder if the Docker instance actually stopped receiving messages (indicating a send side problem), or it they are hitting the server but somehow not being persisted.

**Comment by alangbha** [comment]
yes, I do. Unfortunately I didn't find anything meaningful. However, it would be nice to narrow the search but I don't know in which services/containers is better to search or what kind of error (if known)

**Comment by alangbha** [comment]
Something interesting that just happened. I had this error in snuba outcomes consumer
```
arroyo.errors.OffsetOutOfRange: KafkaError{code=_AUTO_OFFSET_RESET,val=-140,str="fetch failed due to requested offset not available on the broker: Broker: Offset out of range (broker 1)"}
```
I fixed it by resetting offsets from the outcomes topic "to-latest"...as soon as I did it, I saw this in the stats graph
![image](https://github.com/getsentry/self-hosted/assets/94523389/0fecd10e-6b5b-4cf9-afe5-177d8b1371cb)

**Comment by hubertdeng123** [comment]
Gotcha, is that what you expect to see? That `AUTO_OFFSET_RESET` error is fixed in 23.5.0.

Fix: https://github.com/getsentry/self-hosted/pull/2144

**Comment by alangbha** [comment]
I tried to upgrade but I saw there would be some issues with Postgresql version and that's why I postponed it. I'll come back when I have updates. Thanks!
